sensor/kubernetes/complianceoperator/updater_test.go

```diff
@@ -0,0 +1,179 @@
+package complianceoperator
```

Comment:
Expanding this unit test to cover the CRDs cannot be done with current kubernetes client used in Sensor because we need to register the custom scheme used by CO CRDs.